tutorials/observing_examples.ipynb

```diff
@@ -0,0 +1,441 @@
c2dGyVl1ACCZqC3OzDlnl5DT/YoWEtYhnhePHd4xzZl5/Ungv9cCPWtah53LNtNtSwwsmZOw/jvxziahLpCKwJe15hLwt3J3ChiFRg3YVc42I8gbKrsorfv7mEEfe9x4/DmuB+etvJcbdKSgU9S5t+pehmT3s3uXpS8kjtXDFBHR5kbP92nHZU9I6x/7io/vHJbhnXp0n79dsdHLibRKJdY0T+BcYDjxtjOgGnAU+IyGExicgVIjJDRGZs2rQp8uWU80XFDk594EP+8t6yOifPZb861ddjTSVbvH+Kbw6KvIZRXrn3W0dxwym9efeGkV6HkvAdbX2zajY0GdXRnWKf5iAaH5ZmuVqxXgGE9+7qxOHFVZcB4wCMMdNEJBcoBurUuhljHgYeBigvL/fhn9E5by/cwDVPz6ZNi2a88OP45yVPtmR1unIid2ZqAvaNopY5XOODGRbBuqtIxDWje/HHt5c6HE3DmlCd4ho3k8h0oJeIdAPWAucD34tYZzVwMvC4iPQFcoHUv9WoxxPTVvGLSQs4qmMhj1w8hJI4e2Qn2+K7x5HlYaG23pWppvr56X25dHi3hN7rRcs6P85u6FpxljGmGrgamAIswmqFtUBE7hKRs+zVbgB+KCJzgaeBS4wf/0pJ8Mb8ddz+ygJG92nLM1cM830CAausPlmDEEY2wXz04vL4t6E5J629FzHnDVijOASpmbUfT46uNuux+3xMjlh2R9jjhcBwN2Pwu8qqGt5bvJEbnp/LwM6t+Mv3BqVkRWpTRSaAk/t6M0NfkHVsld6jAQQoV9TLj5fYwWwakSLeWbSBEfe9x5VPzqIwL5uHvz9YE4iL0r34K3IiMb947doTHNuW3+d4aXoO8F8WSf0OBj4UChnunbKYh6auoE+7fO4550iGlrWhtQPzMKeqwrzD+4mo+Hg3q2HDSvObPhxIrcaKpl65ajhfrN3Bz1+eDwSviFPvRBShkOG2/37BQ1NXcMGxXXjl6uF8o387TSCNuODYLgcfezmC7ohexQzs3Mqz/aumGdC5FRcGeLgcH+YQvRNJplDIcOt/5/HcjAquGd2T60/pnfZFLLEKr8C/86z+nsXxxGXHerZv5bzTj27vdQhx8eOwJ5pEkiQUMtzy4jyen1nBtSf34roxvTSBKOUxp8ePm/HzMY5uL1K8A0UmgyaRJDDGcM9riw4mkOtP6e11SEopFzgxUm/QaJ1IEvzt/eX88+OVXDq8jOvG+KOXrlJ+4eQNud7cJ58mEZdNmvs1v52yhHMGduD20/tpEZZSEZws5j+mi7+HCUpFWpzlotmrt3Hj83MZWtaGe8872rdNLIPinnOOpCjJrdg6t8ljzdZ9Sd2nSsytp/bhxN4lXofRIB/WizeZJhGXrN2+jx/+eyZtC3J48PuDAzUBlF950TTzf1efwMZd+xl7/wdJ37cT+rUvYOG6nV6HkRStm1t9iabeNJKszAyGT3zX44jSgyYRF+zZX83l/5rB/qoanv7hsbTRPiCB1ap5M1o1188vSLoWtfA6hLSidSIOM8Zw8wvzWLJ+J3/+3iB61TPngFLJMKqPv4t3VPBpEnHY45+s4rUv1nHzuD6MPKLU63BUijiyY0HjK0Vx1gD/T8Tlx9n6/GrvgWqvQziMJhEHzfxqG796bRFj+rblRyd29zqcwMkJ6FSpyfDcj4Yl9L4j2umdcCppV+jcOGNO0f9ah2zfe4Crn5pF+1a5/P47A7QpbwLOHNDB6xB8q3ZO8kR9e3AnhyJRXvJj6y6tWHeAMYYJL89n0679vPST4TrirPKVxXePo1mSJg/zqw6Fub68ik8F6f3Ncsgrc77mtXnruO6U3hzVqdDrcALrypE9vA4hJeVmZ6ZFH6WGxpX65LaTfdHMvqn1P7e8OM+hSJyjSaSJ1m7fx+2vzKe8a2t+fJKeBJuiR0lLr0NQHvDjoIJ+tW5HpdchHEaTSBMYY7jxubmEQob7vzswUHM1K//oVpze/Rrcap01qIvO+5IMWifSBC/OWsu0FVv4zblH0blNc6/DUQHVqXUeKzfv8TqMlJPdQD3Q45cOYdveA0mMJnVpEknQjr1V/GbyIo7p0orvlnf2OhylVBy0D5dzNIkk6LdvLmbb3gM8cdmxaVFpqdzToTCv3tfOCNjMe37UqXX9f1/VdFonkoC5a7bz5Gerufj4Mvp1SKwnsVK1MsL+C4tb1h2na5AObd5kvz1vgNchHFSSgpNWaRKJU03IcPsr8ylumcN1OkOhckB4B7LImfH6pkOPc6fq1espEPBTv99ebfN9FY8TtDgrTk9/vpp5FTt44PyBFORqp0LljteuPYGqGsPAztrCKNUc1bGQeRU7vA7DMZpE4rB5937ue2Mxw7oXcZYO0aFclCHCwM7acVX5nxZnxWHi64vZV1XD3ef017GxlFIKTSIxm75qKy/MrODyEd3pWZoG5dQq8P5xUbnXISTVYZd1PhysMBVpEolBdU2I21+eT8dWeVwzuqfX4SgVk9L81GsJ1JD6cobfygz8OBJvU2gSicHjn6xi8fpd3HFmvyYPya2UUqlEz4iN2LCzkvvf+pJRR5Qwtl9br8NRHnn1mhOYvWa712HEpX+a9WHy4o7jzAEd+N/cr+N6T7KrU6ct38KabXtd276rdyIiMk5ElojIMhG5tZ51viMiC0VkgYg85WY8iXjgnaUcqAlx51lamZ7OjuxYyPeP6+r6fpws6shK8zlEkuHP4wd5HUKjxv/jU25+wb0h5F27ExGRTOCvwClABTBdRCYZYxaGrdMLuA0YbozZJiK+GtBm9Za9PDd9Dd87tgtdi9J7pFXlnmjDcuj1igoKNy9VhgLLjDErjDEHgGeAsyPW+SHwV2PMNgBjzEYX44nbA+8sJTNDuGqUVqYr91w5Ur9f6eiEnsVeh+AIN5NIR2BN2PMKe1m43kBvEflYRD4VkXEuxhOXZRt38dLsCi4a1pW2BTqtpnKPG/PQvHrNCY5v0y2OjXqit2+ecD
```

Comment:
Let's make a final cell which compares the synphot expected count rate (y) with the measured count rate (x), with a 1:1 line over plotted.

Reply:
Do you have a paper I could cite for the observed values?

Reply:
The observed values aren't published in papers, but the observations themselves make up two research notes: [HAT-P-11](http://adsabs.harvard.edu/abs/2018RNAAS...2a..26M) and [TRAPPIST-1](http://adsabs.harvard.edu/abs/2018RNAAS...2a..10M).